independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+                      "powerProductionTotal": {
+                        "type": "number",
+                        "description": "Power Production Total Forecast",
+                        "x-ms-summary": "Total Production"
+                      }
+                    },
+                    "type": "object"
+                  },
+                  "type": "array",
+                  "description": "Power Production forecast data",
+                  "x-ms-summary": "Power Production Forecast"
+                },
+                "updatedAt": {
+                  "type": "string",
+                  "description": "Date and time when forecast has been updated",
+                  "x-ms-summary": "Updated At"
+                },
+                "zone": {
+                  "type": "string",
+                  "description": "Country or Zone",
+                  "x-ms-summary": "Zone"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "operationId": "BreakdownForecast",
+        "summary": "Get Power Production Forecast",
+        "description": "Retrieves the forecasted power production breakdown of an area by production type. It can either be queried by zone identifier or by geolocation"
+      }
+    },
+    "/zones": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Zones": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Name": {
+                        "type": "string",
+                        "description": "Name"
+                      },
+                      "Details": {
+                        "type": "object",
+                        "properties": {
+                          "zoneName": {
+                            "type": "string",
+                            "description": "Name of zone",
+                            "x-ms-summary": "Zone Name"
+                          },
+                          "countryName": {
+                            "type": "string",
+                            "description": "Name of country",
+                            "x-ms-summary": "Country Name"
+                          }
+                        },
+                        "description": "Details for Zone",
+                        "x-ms-summary": "Zone Details"
+                      }
+                    }
+                  },
+                  "description": "Zone List",
+                  "x-ms-summary": "Zones"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Available Zones",
+        "description": "A list of countries and zones by code (zone being the country is divided by zones seen on app.electricitymap.org)",
```

Comment:
@VitaliiSorokin all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)